.github/workflows/cd-post-release-tests.yml

```diff
@@ -170,10 +170,10 @@ jobs:
 
       - name: Install Syft
         run: |
-          if (${{ inputs.release_platform }} == "REAL_PYPI"); then
+          if [ "${{ inputs.release_platform }}" == "REAL_PYPI" ]; then
```

Comment:
NITPICK: In shell commands singe square brackets work only -eq or `=` but only with double square brackets `==` operation works

Reply:
Fixed using :https://github.com/OpenMined/PySyft/pull/8649/commits/c4c851f7abfe340f03d28760d0386cbf4f511171